packages/grafana-prometheus/package.json

```diff
@@ -69,6 +81,10 @@
     "@grafana/e2e": "workspace:*",
     "@grafana/e2e-selectors": "workspace:*",
     "@grafana/tsconfig": "^1.3.0-rc1",
+    "@rollup/plugin-commonjs": "25.0.2",
+    "@rollup/plugin-image": "3.0.3",
+    "@rollup/plugin-json": "6.0.0",
```

Comment:
Not sure this is needed. I cannot see it being used in the rollup.config file.

Reply:
Removing!